modules/ROOT/pages/audit-log-retention.adoc

```diff
@@ -0,0 +1,40 @@
+= Audit Log Retention Period
+
+xref:audit-logging.adoc[Audit logs] have a default retention period of six years. Users who have Audit Log Config Manager and Organization Administrator permissions can configure the retention period for the audit logs in their organization.
+
+Download your logs periodically if you want to maintain your log files for longer than six years.
+To check the audit log retention period for your organization, check the retention period in your root organization's settings, or use the https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/minor/1.0/console/method/%23335/[Audit Logging API^].
```

Comment:
I uploaded new docs and so this url is now:
https://anypoint.mulesoft.com/exchange/portals/anypoint-platform/f1e97bc6-315a-4490-82a7-23abe036327a.anypoint-platform/audit-log-query-api/minor/1.0/console/method/%23341/